Automattic/jetpack
# Portfolio: bring changes from WordPress.com to keep files in sync

Fixes #24157

#### Changes proposed in this Pull Request:

* Follow-up to #23848
* The files were not fully in sync. Most of the changes needed to be made on the WordPress.com side, but there are a few things that I think we should bring to Jetpack to keep the 2 files fully in sync from now on.

Internal reference: r124493-wpcom

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Go to Jetpack > Settings > Writing  and activate the Porfolio feature.
* Go to Appearance > Themes > Add New, and install the Illustratr theme
* Go to Portfolio > Add New and create a test project; then another.
* Go to Posts > Add New and in that new post, add a portfolio shortcode, as per [the docs here](https://wordpress.com/support/portfolios/portfolio-shortcode/).
* Ensure the shortcode is displayed properly.
